bin/stacks/routing-dashboard-stack.ts

```diff
@@ -439,10 +440,17 @@ export class RoutingDashboardStack extends cdk.NestedStack {
       },
     ])
 
+    const missingProviderUrlsForChains = SUPPORTED_CHAINS.filter((chainId) => !jsonRpcProviders[`WEB3_RPC_${chainId.toString()}`])
+    const providerUrlsForChains = SUPPORTED_CHAINS.filter((chainId) => !SUPPORTED_CHAINS.includes(chainId))
```

Comment:
This should be `!missingProviderUrlsForChains.includes(chainId)` right?

Reply:
yep good catch. I added the dashboard screenshot to show that the logic is correct now, otherwise no metric should show up.